tilanvarauspalvelu/settings.py

```diff
@@ -0,0 +1,144 @@
+import os
```

Comment:
Hmm, looks like something is wrong here, this overrides stuff that should be in development, maybe something wrong with some merge from dev or something.

Reply:
I think this hasn't been rebased after changing name of project and project folder now appears as new becaus it has different name. Please rebase to current develop and push this branch again @otiala

Reply:
Oh right, missed that this still has the old folder name.